wikimedia/wikipedia-ios
# Remove "Go to" prefix from `NotificationCenterAction` text

**Phabricator:** https://phabricator.wikimedia.org/T302453#7829975 (partially)

### Notes
Removes the "Go to" prefix text from `NotificationCenterAction` cells.

### Test Steps
Using the Notifications Center > Cell Swipe > More Actions sheet and tapping a notification for the detail view, confirm the "Go to" prefix text is no longer present on action titles.
